### PR TITLE
ios add Order History

### PR DESCRIPTION
**Pull request by diarmuidmcg**

### CODE REVIEW

**Comment by diarmuidmcg** [comment]
* **#135** <a href="https://app.graphite.dev/github/pr/miamtech/mealz-documentation/135?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#134** <a href="https://app.graphite.dev/github/pr/miamtech/mealz-documentation/134?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/miamtech/mealz-documentation/134?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `develop`

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: docs/mobile/ios/features/_shared/_components/BaseHeaderComponent.mdx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+```swift
+public struct BaseHeaderParameters {
+    /// The title of the page
+    public let title: String
+    /// Callback to return to the OrderHistory page
+    public let goBack: () -> Void
+```
+
+[//]: # (- [BaseHeaderProtocol]&#40;https://miamtech.github.io/MealziOSSDKRelease/documentation/mealziossdk/baseheaderprotocol&#41;)
```
Comment (antonin-francois) [inline docs/mobile/ios/features/_shared/_components/BaseHeaderComponent.mdx:80]: c'est quoi cette dernière ligne là ?

File: docs/mobile/ios/features/order-history/_custom-views/OrderDetailsViewOptions.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#### Order Details - `OrderDetailsViewOptions`
+
+```swift
+import MealziOSSDK
+
+static let orderDetailsViewOptions = OrderDetailsViewOptions(
+    orderDetailsHeader: TypeSafeBaseHeader(/* your new view*/),
+    orderDetailsRecipeCard = TypeSafeOrderDetailsRecipeCard(/* your new view*/)
```
Comment (antonin-francois) [inline docs/mobile/ios/features/order-history/_custom-views/OrderDetailsViewOptions.md:8]:
```suggestion
    orderDetailsHeader: TypeSafeBaseHeader(/* your new view */),
    orderDetailsRecipeCard = TypeSafeOrderDetailsRecipeCard(/* your new view */)
```

File: docs/mobile/ios/features/order-history/_custom-views/OrderHistoryViewOptions.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+#### Order History - `OrderHistoryViewOptions`
+
+```swift
+import MealziOSSDK
+
+static let orderHistoryViewOptions = OrderHistoryViewOptions(
+    searchBar: TypeSafeSearch(/* your new view*/),
+    previousOrderCard = TypeSafePreviousOrderCard(/* your new view*/)
```
Comment (antonin-francois) [inline docs/mobile/ios/features/order-history/_custom-views/OrderHistoryViewOptions.md:8]:
```suggestion
    searchBar: TypeSafeSearch(/* your new view */),
    previousOrderCard = TypeSafePreviousOrderCard(/* your new view */)
```

File: docs/mobile/ios/features/_shared/_components/BaseHeaderComponent.mdx (hunk 1 above)
Comment (antonin-francois) [reply docs/mobile/ios/features/_shared/_components/BaseHeaderComponent.mdx:80]:
```suggestion
```

**Review by antonin-francois — APPROVED** [review]

**Pull request merged**